src/background/modules/ContextMenu.js

```diff
@@ -12,6 +11,38 @@ const PREVIEW_STRING_CUT_LENGTH = 100; // a setting that may improve performance
 let lastCachedUnicodeFontSettings = null;
 let menuIsShown = false;
 
+let pasteSymbol = null;
+
+/**
+ * Create notification.
+ *
+ * @param {string} title
+ * @param {string} message
+ * @returns {void}
+ */
+function notification(title, message) {
```

Comment:
You may wanna translate the error message/make it translatable.

I suggest to use a module like this:
https://github.com/rugk/mastodon-simplified-federation/blob/2c384be340b64d8512bd3f0d6082c38ebb75136a/src/common/modules/Notifications.js#L23

Either copy and pate or of course we may want to move it to https://github.com/TinyWebEx/.

Reply:
OK, good idea. Will do...